ocd_backend/items/a2a.py

```diff
@@ -0,0 +1,414 @@
+from datetime import datetime
+from pprint import pprint
+
+from ocd_backend.log import get_source_logger
+from ocd_backend.items import BaseItem
+
+log = get_source_logger('loader')

```

Comment:
Do note that there are other logger wrongly initialized then as well (openbeelden for example)